vcpkg/commands/common-options.md

```diff
@@ -111,7 +111,7 @@ Defaults to searching upwards from the current working directory for the nearest
 
 ### <a name="overlay-ports"></a> `--overlay-ports=<path>`
 
-Specifies a directory containing [overlay ports](../concepts/package-name-resolution.md#overlays).
+Specifies a directory either an [overlay port](../concepts/package-name-resolution.md#overlays), or subdirectories that are themselves overlay-ports. If `<path>` contains a `CONTROL` or `vcpkg.json`, subdirectories are not considered.
```

Comment:
```suggestion
Specifies a directory either an [overlay port](../concepts/package-name-resolution.md#overlays), or subdirectories that are themselves overlay-ports. If `<path>` is a folder containing a `CONTROL` or `vcpkg.json`, it is treated as a overlay port folder and subdirectories will **not** be considered as additional overlay ports.
```

I find a slightly longer explanation better since the short versions raises the question: "For what are subdirectories not considered?"

Reply:
I applied this suggestion, except I kept 'directory' rather than 'folder', and added a comma.